docs/docs/pods.md

```diff
@@ -0,0 +1,583 @@
+---
+title: Pods
+---
+
+# Pods
+
+Marathon supports the creation and management of pods. Pods enable you to share storage, networking, and other resources among a group of applications on a single agent and address them as one group rather than as separate applications, similar to a virtual machine. Pods allow quick, convenient coordination between applications that need to work together, for instance a database and web server that make up a content management system.
+
+Currently, Marathon pods can only be created and administered via the `/v2/pods/` endpoint of the REST API, not via the web interface.
+
+# Features
+
+- Co-located containers.
+- Pod-level resource isolation.
+- Sandbox and ephemeral volumes that are shared among pod containers.
+- Configurable sandbox mount point for each container.
+- Pod-level health checks.
+
+# Quick Start
+
+1. Run the following REST call, substituting your IP and port for `<ip>` and `<port>`:
+
+    ```bash
+    $ http POST <ip>:<port>/v2/pods <<EOF
+    > {
+    >   "id": "/simplepod",
+    >   "scaling": { "kind": "fixed", "instances": 1 },
+    >   "containers": [
+    >     {
+    >       "name": "sleep1",
+    >       "exec": { "command": { "shell": "sleep 1000" } },
+    >       "resources": { "cpus": 0.1, "mem": 32 }
+    >     }
+    >   ],
+    >   "networks": [ {"mode": "host"} ]
+    > }
+    > EOF
+    ```
+
+1. Verify the status of your new pod:
```

Comment:
it would be useful to call attention to the pod id.   the pod id is used for all interaction with the pod as soon as the pod is created.